paint/preprocessing/calibration_stac.py

```diff
@@ -16,7 +16,7 @@ def make_calibration_collection(
     heliostat_id: str
         The heliostat id of the heliostat being considered.
     data: pd.DataFrame
-        The dataframe containing all image data.
+        The dataframe containing all image preprocessing.
```

Comment:
Change "preprocessing" back to "data" (cannot make commit suggestion).

Also add spaces before colons for each parameter, e.g.:
`heliostat_id : str`